Issue 120 - Improved readme file (#121)
Cognifide/AEM-Rules-for-SonarQube: README.md

3. Restart SonarQube.
4. Go to rules section and activate AEM rules in your profile.

## Usage

> Usage of the plugin does not differ much from a regular SonarQube analysis. However, as rules are more often tied to certain AEM version and its components (Felix, Sling) we've introduced `aemVersion` analysis property. 

Reviewer: ```suggestion
Use of the plugin does not differ much from regular SonarQube analysis. However, as rules are often tied to a certain AEM version and its components (Felix, Sling), we've introduced the `aemVersion` analysis property.
```

Each rule defines supported AEM version or version range. Most of the rules are universal.

> By providing AEM version parameter, you're informing Sonar Runner to use only subset of rules supporting particular AEM version. When the parameter is not provided then a default AEM version is used (currently 6.4)

Reviewer: ```suggestion
By providing the AEM version parameter, you can instruct the Sonar Runner to only use only a subset of rules applicable to a particular AEM version. When the parameter is not provided then a default AEM version is used (currently 6.4)
```
